src/main/java/ch/uzh/ifi/hase/soprafs22/game/enums/Underground.java

```diff
@@ -0,0 +1,5 @@
+package ch.uzh.ifi.hase.soprafs22.game.enums;
+
+public enum Underground {
+    WATER, GRASS, RUBBLE, BRIDGE, ROAD
```

Comment:
We maybe need to create a class of tile, because there are around 39 different tiles, ergo 39 enums. And some of them are just variations, like bank left, bank top left corner, etc.. Additionally we could save if they are accessible, this however would also be possible for enums.

Reply:
thx!
added a tile class, so you'll know how I meant the underground enum. a tile is composed of many more attributes like slope, position, etc., Underground is just one of them ;)